www/apps/book/app/advanced-development/modules/module-links/page.mdx

```diff
@@ -38,13 +38,13 @@ import { defineLink } from "@medusajs/utils"
 
 export default defineLink(
   HelloModule.links.myCustom,
```

Comment:
@olivermrbl am I correct in assuming the data model's name here is the camel case name? For example, this data model is actually defined as:

```
import { model } from "@medusajs/utils"

const MyCustom = model.define("my_custom", {
  id: model.id().primaryKey(),
  name: model.text(),
})

export default MyCustom
```

Reply:
Yes, that should be the case. Looping in @adrien2p to verify